src/pluralkit/bot/commands/misc_commands.py

```diff
@@ -158,8 +158,13 @@ def make_command_list(lst):
     await working_msg.delete()
 
     f = io.BytesIO(json.dumps(data).encode("utf-8"))
-    await ctx.reply_ok("DM'd!")
-    await ctx.message.author.send(content="Here you go!", file=discord.File(fp=f, filename="pluralkit_system.json"))
+    export_msg = ctx.message.author.send(content="Here you go! \u2709", file=discord.File(fp=f, filename="pluralkit_system.json"))
+
+    if isinstance(ctx.message.channel, discord.DMChannel):
+        await export_msg
+    else:
+        await ctx.reply_ok("DM'd!")
```

Comment:
The ordering doesn't matter, you don't need this weird Future concoction.

Reply:
*wheeze*